htdocs/societe/class/societe.class.php

```diff
@@ -462,6 +485,12 @@ class Societe extends CommonObject
 	 */
 	public $idprof6;
 
+	/**
+	 * Object of company
+	 * @var string
+	 */
+	public $object;
```

Comment:
Objet of company inside company ? Strange

Reply:
https://github.com/Dolibarr/dolibarr/blob/dd4a9a94c7fbb8daa745e3a2ff880f3871be8206/htdocs/societe/class/societe.class.php#L4271

Reply:
https://github.com/Dolibarr/dolibarr/commit/b6e333876ed5fd371e8e51aae8a52585d1bd506c

Reply:
Hum i see it is not the "object" in POO sense but into the meaning "Reason/Goal of the company".